src/main/__tests__/startPasswordless.spec.ts

```diff
@@ -16,7 +16,7 @@ beforeEach(() => {
   fetchMock.resetMocks()
 })
 
-const authParams: SingleFactorPasswordlessParams = { authType: 'magic_link', email: 'john.doe@example.com' }
+const authParams: PasswordlessParams = { authType: 'magic_link', email: 'john.doe@example.com' }
```

Comment:
Can move these changes to separate PR pls ?

Reply:
@StanWarz said he'd include this fix in a nother PR, but for now I need it on this branch so that reachfive-sandbox can run.